File: server/ITWarsEngine/src/EconomicPlanet.java
public class EconomicPlanet extends Planet implements Cloneable {

	public int revenue;

	public EconomicPlanet(int owner, int numShips, int growthRate, double x,
			double y) {
		this.owner = owner;
		this.numShips = numShips;
		this.revenue = growthRate;
		this.x = x;
		this.y = y;
	}

	private EconomicPlanet(EconomicPlanet pSP) {
		this.owner = pSP.owner;
		this.numShips = pSP.numShips;
		this.revenue = pSP.revenue;
		this.x = pSP.x;
		this.y = pSP.y;
	}

	public Object clone() {
		return new EconomicPlanet(this);
	}

}

File: starters/cSharp/EconomicPlanet.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace DTStrike.MyBot
{
    public class EconomicPlanet : Planet
    {
        public int revenue;

        public EconomicPlanet(int id, int owner, int numShips, int revenue,
                double x, double y) : base(id, owner, numShips, x, y)
        {
            
            this.revenue = revenue;
        }
    }
}
